driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/pageobjects/internal/action/GuiElementCheckFieldAction.java

```diff
@@ -70,13 +73,34 @@ private void pCheckField(UiElementBase guiElement, Check check) {
                 }
             }
         }
-
         int prevTimeout = -1;
-
         if (useTimeout >= 0) {
             prevTimeout = overrides.setTimeout(useTimeout);
         }
 
+        /**
```

Comment:
JavaDoc ineffective at this level. Needs to be put atop a type or method declaration. (Did you mean `/*`?)